test/iptables/nat.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iptables
+
+import (
+        "fmt"
+        "net"
+)
+
+const (
+        redirectPort     = 42
+)
+
+func init() {
+        RegisterTestCase(FilterNATRedirectUDPPort{})
```

Comment:
Nit: indentation is slightly off in a bunch of places. It looks like your editor is sometimes using spaces and sometimes using tabs.